packages/mui-material/src/Tabs/Tabs.js

```diff
@@ -443,10 +443,34 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
     scroll(scrollValue);
   };
 
+  const getFirstVisibleTab = (tabs) => {
+    const containerSize = vertical ? tabsRef.current.clientHeight : tabsRef.current[clientSize];
```

Comment:
```suggestion
    const containerSize = tabsRef.current[clientSize];
```

This is the identical (`clientSize` is being set based on the `vertical` prop).